### ISSUE TITLE
one crash

### ISSUE THREAD

**Issue by rcen** [Microsoft/PTVS#3927]
[Window Title]
devenv.exe

[Main Instruction]
An unexpected error occurred

[Content]
Please press Ctrl+C to copy the contents of this dialog and report this error to our issue tracker.

[^] Hide details  [Close]

[Expanded Information]
```
Build: 15.6.18045.1

System.ArgumentNullException: Value cannot be null.
Parameter name: key
   at System.Collections.Concurrent.ConcurrentDictionary`2.TryRemove(TKey key, TValue& value)
   at Microsoft.PythonTools.Intellisense.VsProjectAnalyzer.<UnloadFileAsync>d__139.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.Infrastructure.VSTaskExtensions.<HandleAllExceptions>d__5.MoveNext()
```

**Comment by zooba** [comment]
Thanks! This has been fixed for 15.7, but we may want to consider for 15.6? @huguesv

**Comment by huguesv** [comment]
Sure, that's 2 external reports, let's add it to 15.6.
https://github.com/Microsoft/PTVS/issues/3826

**Comment by huguesv** [comment]
Won't be fixed for 15.6.
